src/components/templates/UiQuestion/UiQuestion.vue

```diff
@@ -165,10 +167,13 @@ export interface Questionsettings {
   info: boolean;
   why: boolean;
   issue: {
+    action: boolean;
```

Comment:
Please also rename the interface to `QuestionSettings`. 😅

Reply:
:thank_you: